lib/testing/src/lib/core/utils/string.util.ts

```diff
@@ -107,18 +107,20 @@ export class StringUtil {
         return fileNames;
     }
 
-    /**
-     * Generates a random name for a process
+    /** Generates a random name for a process
+     *
+     * @param length  {int} If this parameter is not provided the length is set to 5 by default.
      */
-    static generateProcessName(): string {
-        return 'process_' + StringUtil.generateRandomString(5);
+    static generateProcessName(length: number = 5): string {
+        return 'process_' + this.generateRandomString(length);
```

Comment:
is a static class I think StringUtil. was the right use

Reply:
I've changed back to StringUtil. Thanks